reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -722,3 +722,46 @@ def _verify_k8s_version(self):
                           .format(cur=curr_ver, expected=expected_ver))
 
         return k8s_version_compatibility
+
+    def get_pod_status(self, component=None):
+        """Return status for pods in cluster."""
+        if component and component.startswith('reana-'):
+            component = component.replace('reana-', '')
+        all_pods = self._corev1api.list_namespaced_pod('default')
+        pod_status = dict()
+
+        if component:
+            for pod in all_pods.items:
```

Comment:
The behavior which determines the status of a component can be moved to a private function  so you can make this function smaller. Besides just cosmetic comment, I would name the function `get_components_status`, first replacing `pod` with component (pod is just the implementation of a component in k8s) and I would make it plural since the default behavior is getting all components statuses.

Reply:
I agree, I pushed the changes you mention